modules/tiles/test/tileset/helpers/get-frame-state.spec.js

```diff
@@ -50,17 +50,10 @@ test('getFrameState', t => {
   const results = getFrameState(viewport, 1);
   t.equals(results.height, expected.height, 'height should match.');
   t.equals(results.frameNumber, expected.frameNumber, 'frameNumber should match.');
-  /* TODO 
   t.ok(
     equals(results.camera.position, expected.camera.position, EPSILON),
     'camera.position should match.'
   );
-  t.ok(
```

Comment:
Are these removed? Do we not need them? Should we replace with tests for the fields we use?

Reply:
Done. Changed test + added one more with fields we use.